dll/shellext/netplwiz/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+
+spec2def(netplwiz.dll netplwiz.spec ADD_IMPORTLIB)
+
+list(APPEND SOURCE
+    netplwiz.c
+    SHDisconnectNetDrives.c
+    ${CMAKE_CURRENT_BINARY_DIR}/netplwiz_stubs.c)
```

Comment:
Please move this generated file to the `add_library` instead of in the `SOURCES`